.github/workflows/build-test-windows.yml

```diff
@@ -19,7 +19,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  windows-unittest:
+  windows-2019-unittest:
```

Comment:
Not sure where I stand on it yet, but I at least wanted to bring up the discussion on how we want to handle this going forward. Pre-2022 Windows Server, this bug exists and we have no guarantee that they'll patch it in existing OS versions. So long as we support them, we must ensure that we don't have a regression, but I think ideally we would want to phase this out to just handle the fixed implementation that Windows Server 2022 has now.